### ISSUE TITLE
company-irony's complete speed vs company-clang's

### ISSUE THREAD

**Issue by Voleking** [Sarcasm/company-irony#33]
I use company-clang to complete but I find it's very slow (2s+) and I find answer in this [issues](https://github.com/company-mode/company-mode/issues/592)

> company-clang uses the clang program for completion, and it's slow in nontrivial C++ projects (C++ is generally hard to parse, the standard library is larger, etc).

As one of the alternative backends, I wonder the company-irony complete speed.

**Comment by Sarcasm** [comment]
It all depends on what you are completing, but company-irony should be faster.

**Comment by Voleking** [comment]
As a novice, just standard library. Thx you reply, I will have a try.
